draft-ietf-webtrans-http2.md

```diff
@@ -497,14 +497,24 @@ part of a WebTransport stream.
 
 After sending a WT_RESET_STREAM capsule, an endpoint ceases transmission of
 WT_STREAM capsules on the identified stream. A receiver of a WT_RESET_STREAM
-capsule can discard any data that it already received on that stream.
+capsule can discard any data in excess of the Reliable Size indicated, even if
+that data was already received.
+
+The WT_RESET_STREAM capsule follows the design of the QUIC RESET_STREAM_AT frame
```

Comment:
nit: should we rename `WT_RESET_STREAM` to `WT_RESET_STREAM_AT` to avoid any confusion?

Reply:
I considered that and concluded that the _AT is a quirk we don't need to retain, but I'm happy to do that if someone wants to make an argument in favor of the longer name.

Reply:
FWIW I don't feel very strongly, happy to have you decide